docs/README.md

```diff
@@ -33,4 +33,9 @@ This was last updated on **{{ new Date($page.lastUpdated).toLocaleDateString('en
 Hello world!
 
 &mdash; [Carding](/roster/carding.md)
+
+::: tip April 20, 2020
+[Message Here]
```

Comment:
I think you might've forgotten to actually add in a message here :p

Reply:
hahahaha yeah I did forget